.github/actions/integration-tests/base

```diff
@@ -30,3 +30,29 @@ function log_rc {
         echo "rc=${rc}"
     fi
 }
+
+function validate_ci_run_environment {
+    case "${CI_RUN_ENVIRONMENT}" in
+        standalone|github)
+            echo "CI Run Environment is '${CI_RUN_ENVIRONMENT}'"
+            echo
+            ;;
+        *)
+            echo "ERROR: Unknown value for --run-environment [${CI_RUN_ENVIRONMENT}].  Acceptable values are 'standalone' and 'github'."
+            exit 1
+            ;;
+    esac
+}
+
+function validate_ci_endpoint {
+    case "${CI_ENDPOINT}" in
+        remotehost|k8s)
+            echo "CI Endpoint is '${CI_ENDPOINT}'"
+            echo
+            ;;
+        *)
+            echo "ERROR: Unknown value for --ci-endpoint [${CI_ENDPOINT}].  Acceptable values are 'remotehost' and 'k8s'."
+            exit 1
```

Comment:
not a problem, just not ideal for creating testcases.... if you want to check the desired behavior, you may want to have separate exit codes for each scenario